apps/freelance/src/utils/validations/editEmployerSchema.ts

```diff
@@ -3,20 +3,36 @@ import { IEditEmployerForm } from "@freelance/components";
 import { useJobsValidationErrorMessages } from "utils/constants/jobs-validation-error-messages";
 
 export const useEditEmployerSchema = (): SchemaOf<IEditEmployerForm> => {
-	const { FIELD_REQUIRED, MESSAGE_JOB_TITLE_MAX_CHAR, CORRECT_PHONE, MIN_TWELVE_CHAR } =
-		useJobsValidationErrorMessages();
+	const {
+		FIELD_REQUIRED,
+		MESSAGE_JOB_TITLE_MAX_CHAR,
+		CORRECT_PHONE,
+		MIN_TWELVE_CHAR,
+		MESSAGE_DESCRIPTION_MAX_CHAR,
+		WEBSITE_ERROR_MESSAGE,
+		LINKED_IN_ERROR_MESSAGE,
+	} = useJobsValidationErrorMessages();
 
 	return object({
 		fullName: string().max(50, MESSAGE_JOB_TITLE_MAX_CHAR).required(FIELD_REQUIRED),
 		position: string().max(50, MESSAGE_JOB_TITLE_MAX_CHAR).required(FIELD_REQUIRED),
 		companyName: string().max(50, MESSAGE_JOB_TITLE_MAX_CHAR).required(FIELD_REQUIRED),
 		phone: string()
+			.required(FIELD_REQUIRED)
 			.max(50, MESSAGE_JOB_TITLE_MAX_CHAR)
 			.min(12, MIN_TWELVE_CHAR)
-			.matches(/^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/, CORRECT_PHONE)
-			.required(FIELD_REQUIRED),
-		linkedIn: string().max(50, MESSAGE_JOB_TITLE_MAX_CHAR).required(FIELD_REQUIRED),
-		website: string().max(50, MESSAGE_JOB_TITLE_MAX_CHAR).required(FIELD_REQUIRED),
-		aboutUs: string().max(50, MESSAGE_JOB_TITLE_MAX_CHAR).required(FIELD_REQUIRED),
+			.matches(/^[\\+]?[(]?[0-9]{3}[)]?[-\s\\.]?[0-9]{3}[-\s\\.]?[0-9]{4,6}/gm, CORRECT_PHONE),
+		linkedIn: string()
+			.max(50, MESSAGE_JOB_TITLE_MAX_CHAR)
+			.required(FIELD_REQUIRED)
+			.matches(
+				/^(http(s)?:\/\/)?([\w]+\.)?linkedin\.com\/(pub|in|profile)/gm,
+				LINKED_IN_ERROR_MESSAGE,
+			),
+		website: string()
+			.max(50, MESSAGE_JOB_TITLE_MAX_CHAR)
```

Comment:
why you duplicate 50 here and on the text? imagine you need change validation from 50 to 80, you will need to change it in 2 places, and don't forget that you have 2 places, so hight chance that you will have a bug, you should have one variable that will be responsible for text and validation itself

Reply:
fixed